config/routes.rb

```diff
@@ -5,6 +5,7 @@
   devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root to: 'home#dashboard'
+  get 'search', to: 'users#search'
```

Comment:
why is this line added when the same is done in line 14?

Reply:
We need this route if we try to reload the search page. Not sure why it is not working with the one defined in line 14 though, to be honest.

Reply:
did you try using the path helper `search_users_path` as mentioned below it is not just `/search` but `users/search` and due to that the correct path is `search_users_path`

Reply:
Now it works, thanks 😄 .